apps/mull-ui/src/utilities.ts

```diff
@@ -83,6 +84,16 @@ export const hasEmoji = (text: string) => {
   return !!emojiRegexRGI().exec(text);
 };
 
+const textLength = string.length;
+export const exceedsNbOfLines = (text: string) => {
+  let countLines = 1;
+  for (let i = 0; i < textLength; i++) if (text.charAt(i) === '\n') countLines++;
```

Comment:
Not used to seeing for loops like this, I prefer the usual way even if it's just one line inside. My brain had to pause for a second lool
```suggestion
   for (let i = 0; i < textLength; i++) {
    if (text.charAt(i) === '\n') countLines++;
  }
```